catalog/output/workflows.json

```diff
@@ -34,19 +34,19 @@
           "gtf": "{{ gene_model_url }}"
         },
         "ploidy": "ANY",
-        "trsId": "#workflow/github.com/iwc-workflows/fastq-to-matrix-10x/scrna-seq-fastq-to-matrix-10x-cellplex/versions/v0.6",
-        "workflowDescription": "This workflow processes the CMO fastqs with CITE-seq-Count and include the translation step required for cellPlex processing. In parallel it processes the Gene Expresion fastqs with STARsolo, filter cells with DropletUtils and reformat all outputs to be easily used by the function 'Read10X' from Seurat.",
-        "workflowName": "scRNA-seq_preprocessing_10X_cellPlex"
+        "trsId": "#workflow/github.com/iwc-workflows/fastq-to-matrix-10x/scrna-seq-fastq-to-matrix-10x-cellplex/versions/v0.6.2",
```

Comment:
This is what a normal update will look like, since the current diff for the whole PR is hard to read.